tests/unit/nupic/algorithms/cla_classifier_test.py

```diff
@@ -238,6 +241,53 @@ def testSerialization(self):
     self.assertAlmostEqual(result[1][5], 0.87699877, places=5)
 
 
+  def testWriteRead(self):
+    c1 = CLAClassifier([1], 0.1, 0.1, 0)
+
+    # Create a vector of input bit indices
+    input1 = [1, 5, 9]
+    result = c1.compute(recordNum=0,
+                        patternNZ=input1,
+                        classification={'bucketIdx': 4, 'actValue': 34.7},
```

Comment:
> nit: double quotes

This file uses only single quotes (check `testMissingRecords`, for example). So I think that if we use double quotes here, then we should fix the other occurrences..